bot.js

```diff
@@ -12,19 +13,35 @@ const args = process.argv.slice(2);
 //    console.error("Missing access token.")
 //    process.exit(1);
 //}
-if (args.length != 1 && !process.env.REDDIT_SESSION) {
-    console.error("Missing reddit_session cookie.")
+
+
+let redditSessionCookies = (process.env.REDDIT_SESSION || args[0])
+if (redditSessionCookies) redditSessionCookies = redditSessionCookies.split(';')
+
+let userNames = (process.env.BOT_USERNAME || false)
+if (userNames) userNames = userNames.split(';');
+let passWords = (process.env.BOT_PASSWORD || false)
+if (passWords) passWords = passWords.split(';');
+
+if (userNames || passWords) {
+    if (userNames?.length !== passWords?.length) {
+        console.error('Een user per password graar (en ook andersom).');
```

Comment:
Graar?

Reply:
als in liefst als in dat zou ik willen (https://www.encyclo.nl/begrip/graag)

Reply:
Moet er wel `graag` staan 😜

Reply:
O jemig ik ben blind (sorry heb ook dyslexie)